events/wc_dark_cult_events.txt

```diff
@@ -889,16 +888,27 @@ narrative_event = {
 	border = GFX_event_narrative_frame_intrigue
 
 	is_triggered_only = yes
-	
+
 	immediate = {
 		add_trait = travelling
 	}
 
 	# I will fulfill the will of the Gods!
 	option = {
 		name = EVTOPTA_WCDAC_5030
-
-		hidden_effect = { 73 = { province_event = { id = WCDAC.5034 days = 60 } } }
+		custom_tooltip = { text = chogall_journey_length }
+		hidden_effect = { remove_old_gods_follower_modifier_effect = yes }
+		add_character_modifier = { name = follower_of_cthun duration = -1 }
+		set_global_flag = twilights_hammer_kalimdor
+	}
+	option = {
+		name = EVTOPTB_WCDAC_5030
+		custom_tooltip = { text = chogall_journey_length }
+		hidden_effect = { remove_old_gods_follower_modifier_effect = yes }
+		add_character_modifier = { name = follower_of_yogg_saron duration = -1 }
+	}
```

Comment:
So, I don't see the point of getting the main god before the journey starts when you actually need it at the end.

Reply:
because you hear his whisphers

Reply:
> because you hear his whisphers

So, I hear his whispers but can change the main god sailing.

Reply:
i'll block it then